src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -43,6 +46,29 @@ export const RoomModal: React.FC<RoomModalProps> = ({
   </Modal>
 );
 
+export const RoomModalVideo: React.FC<RoomModalProps> = ({
+  onHide,
+  room,
+  show,
+}) => (
+  <Modal
+    show={show}
+    onHide={onHide}
+    dialogClassName="IntroVideoPreviewModal"
+    centered
+    size="lg"
+    modal
+  >
+    <iframe
+      className="IntroVideoPreviewModal__iframe"
+      title="IntroVideoPreviewModal-iframe"
+      src={room?.url + `?autoplay=1`}
+      allow={IFRAME_ALLOW}
+      allowFullScreen
+    />
+  </Modal>
+);
```

Comment:
[note] Yet here you only show the intro video, with none of the 'normal' `RoomModal` features (including the ability to enter the venue)